python/pyspark/ml/tests.py

```diff
@@ -655,6 +656,20 @@ def test_nested_pipeline_persistence(self):
             except OSError:
                 pass
 
+    def test_write_property(self):
+        lr = LinearRegression(maxIter=1)
+        path = tempfile.mkdtemp()
+        lr_path = path + "/lr"
+        lr.write.save(lr_path)
```

Comment:
Hm, I don't see this discussion, but I figure that testing ```isinstance(lr.write, MLWriter)``` will ensure write is a property in the inheriting class (JavaMLWritable, and also LogisticRegression).  Is that not correct?